sync2jira/main.py

```diff
@@ -128,7 +114,7 @@ def load_config(loader=fedmsg.config.load_config):
     if 'map' not in config['sync2jira']:
         raise ValueError("No sync2jira.map section found in fedmsg.d/ config")
 
-    possible = set(['pagure', 'github'])
+    possible = set(['github'])
```

Comment:
Consider using a "set literal" instead:
```suggestion
    possible = {'github'}
```